sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/specialized/BlobBaseApiTests.java

```diff
@@ -856,4 +880,47 @@ public void accept(BlobQueryError nonFatalError) {
             numErrors++;
         }
     }
+
+    @Test
+    public void abortCopyBaseSimple() {
+        // Data has to be large enough and copied between accounts to give us enough time to abort
+        new SpecializedBlobClientBuilder()
+            .blobClient(bc)
+            .buildBlockBlobClient()
+            .upload(new ByteArrayInputStream(getRandomByteArray(8 * 1024 * 1024)), 8 * 1024 * 1024,
+                true);
+
+        BlobContainerClient cu2 = alternateBlobServiceClient.getBlobContainerClient(generateBlobName());
+        cu2.create();
+        BlobClient bu2 = cu2.getBlobClient(generateBlobName());
+
+        String sas = bc.generateSas(new BlobServiceSasSignatureValues(testResourceNamer.now().plusDays(1),
+            new BlobContainerSasPermission().setReadPermission(true)));
+        SyncPoller<BlobCopyInfo, Void> poller = setPlaybackSyncPollerPollInterval(
+            bu2.beginCopy(bc.getBlobUrl() + "?" + sas, null, null, null, null, null, null));
+        PollResponse<BlobCopyInfo> lastResponse = poller.poll();
+        assertNotNull(lastResponse);
+        assertNotNull(lastResponse.getValue());
+        assertDoesNotThrow(() -> bu2.abortCopyFromUrl(lastResponse.getValue().getCopyId()));
+        // cleanup:
+        // Normal test cleanup will not clean up containers in the alternate account.
+        assertResponseStatusCode(cu2.deleteWithResponse(null, null, null),
+            202);
+    }
+
+    @Test
+    public void syncCopyBaseSimple() {
+        BlockBlobClient bu2 = cc.getBlobClient(generateBlobName()).getBlockBlobClient();
+        String sas = bc.generateSas(new BlobServiceSasSignatureValues(testResourceNamer.now().plusDays(1),
+            new BlobContainerSasPermission().setReadPermission(true)));
+        assertDoesNotThrow(() -> bu2.copyFromUrl(bc.getBlobUrl() + "?" + sas));
+    }
+
+    @Test
+    public void snapshotBase() {
```

Comment:
Instead of using methods on the `BlobClientBase`, could we either just move this test to the `BlobApiTest.java` or migrate this test to one (or all) of the specialized append/page/block clients? Mainly I'm hesitant to be testing against the base class directly as it is more of an implementation detail and users won't be instantiating it directly. By moving it to use one of the public classes that customers will use that gives us a bit more safety in case we decide to refactor this base class or override any of the methods in the client subclasses.